runelite-client/src/main/java/net/runelite/client/plugins/microbot/util/settings/Rs2Settings.java

```diff
@@ -49,6 +50,38 @@ public static void hideRoofs() {
         }
     }
 
+    public static boolean isLevelUpNotificationsEnabled() { return Microbot.getVarbitValue(Varbits.DISABLE_LEVEL_UP_INTERFACE) == 0; }
+
+    public static boolean disableLevelUpNotifications() {
+        if(isLevelUpNotificationsEnabled()){
+            Rs2Tab.switchToSettingsTab();
+            Rs2Widget.clickWidget(7602208);
+            final boolean isSettingsInterfaceVisible = Rs2Widget.getWidget(8781825) != null;
+            sleepUntilOnClientThread(() -> isSettingsInterfaceVisible);
+            if(isSettingsInterfaceVisible){
+                Rs2Widget.clickWidget(8781834);
+                sleep(600);
+                Rs2Keyboard.keyPress('l');
```

Comment:
You can use Rs2Keyboard.typeString to type it in one word

Reply:
should be good now in commit a005a47